src/com/twitter/intellij/pants/ui/PantsToBspProjectAction.java

```diff
@@ -126,13 +127,19 @@ private Path coursierPath() throws IOException {
   }
 
   private List<String> pantsTargets(Project project) {
+    String root = PantsUtil.findBuildRoot(project).map(VirtualFile::getPath).orElse("/");
     PantsSettings pantsSettings = PantsSettings.getInstance(project);
     return pantsSettings.getLinkedProjectsSettings()
       .stream()
       .flatMap(projectSettings -> projectSettings.getSelectedTargetSpecs().stream())
+      .map(target -> stripPrefix(target, root))
       .collect(Collectors.toList());
   }
 
+  private String stripPrefix(String s, String prefix) {
+    return s.startsWith(prefix) ? s.substring(prefix.length()) : s;
```

Comment:
This leaves a leading `/` in the start resulting in project names like `.foo`

Reply:
@olafurpg I implemented removing the `"/"` Hope it works now, as I don't have reproduction for full directory in target names.